pkg/metrics/podlabelmetrics.go

```diff
@@ -1,85 +0,0 @@
-package metrics
-
-import (
-	"github.com/opencost/opencost/pkg/clustercache"
-	"github.com/opencost/opencost/pkg/prom"
-	"github.com/prometheus/client_golang/prometheus"
-)
-
-//--------------------------------------------------------------------------
-//  KubecostPodCollector
-//--------------------------------------------------------------------------
-
-// KubecostPodCollector is a prometheus collector that emits pod metrics
-type KubecostPodLabelsCollector struct {
```

Comment:
No usages

Reply:
I have a feeling we want to leave this in because in the future we want to consider removing dependencies on KSM entirely. At the risk of incredibly annoying product configuration, maybe a third flag to emit these (that is hidden from users)?